src/query/__tests__/RelayQueryRoot-test.js

```diff
@@ -342,6 +342,77 @@ describe('RelayQueryRoot', () => {
     expect(meIdentifyingArg).toBeUndefined();
   });
 
+  it('returns the identifying number argument ', () => {
+    var nQuery = getNode(Relay.QL`
+      query {
+        task(number:5){
+          dumb,
+        }
+      }
+    `);
+    nQuery.getConcreteQueryNode().metadata= {
+      identifyingArgName: 'number',
+      identifyingArgType: 'scalar' };
```

Comment:
@josephsavona nope, that assign `metadata` is something i copyed from a previous exist test. I am not sure what is mean for ...  just because i do not completely know the whole work process of  Relay ,so i copyed some unit test code from some similar exist test.